ui/app/templates/components/identity/edit-form.hbs

```diff
@@ -10,24 +10,39 @@
       {{form-field data-test-field attr=attr model=model}}
     {{/each}}
   </div>
-  <div class="field is-grouped box is-fullwidth is-bottomless">
-    <div class="control">
-      <button type="submit" data-test-identity-submit=true class="button is-primary {{if save.isRunning 'loading'}}" disabled={{save.isRunning}}>
-        {{#if (eq mode "create")}}
-          Create
+
+  <div class="field is-grouped is-grouped-split is-fullwidth box is-bottomless">
+    <div class="field is-grouped">
+      <div class="control">
+        <button type="submit" data-test-identity-submit=true class="button is-primary {{if save.isRunning 'loading'}}" disabled={{save.isRunning}}>
+          {{#if (eq mode "create")}}
+            Create
+          {{else}}
+            Save
+          {{/if}}
+        </button>
+        {{#if (or (eq mode "merge") (eq mode "create" ))}}
+          <a href={{href-to cancelLink}} class="button" data-test-cancel-link>
+            Cancel
+          </a>
         {{else}}
-          Save
+          <a href={{href-to cancelLink model.id "details"}} class="button" data-test-cancel-link>
+            Cancel
+          </a>
         {{/if}}
-      </button>
-      {{#if (or (eq mode "merge") (eq mode "create" ))}}
-        <a href={{href-to cancelLink}} class="button">
-          Cancel
-        </a>
-      {{else}}
-        <a href={{href-to cancelLink model.id "details"}} class="button">
-          Cancel
-        </a>
-      {{/if}}
+      </div>
     </div>
+
+      {{#if (and (eq mode "edit") model.canDelete)}}
+        {{#confirm-action
+          buttonClasses="button is-link is-outlined is-inverted"
```

Comment:
Can we just do `buttonClasses="button is-ghost"` here? I think that's basically what you were going for

Reply:
(sorry for the late response!) ooo yes, that's much nicer - should we also add `has-text-black` or is the blue text ok?

Reply:
Blue text is preferred really. Some places we use black anyway if blue just doesn't look right, but this one needs help making it clear it's an action

Reply:
Oh! Ha, I'll update the policy edit pages then too. 👌